eng/versioning/version_client.txt

```diff
@@ -68,7 +68,7 @@ com.azure:azure-data-schemaregistry-avro;1.0.0-beta.4;1.0.0-beta.5
 com.azure:azure-data-tables;12.0.0-beta.2;12.0.0-beta.3
 com.azure:azure-digitaltwins-core;1.0.1;1.1.0-beta.1
 com.azure:azure-e2e;1.0.0-beta.1;1.0.0-beta.1
-com.azure:azure-identity;1.1.3;1.2.0-beta.3
+com.azure:azure-identity;1.1.3;1.2.0
```

Comment:
We have an unreleased identity entry for version 1.2.0-beta.2. It should be good to remove that?